tests/testthat/test-subset.R

```diff
@@ -102,3 +102,11 @@ test_that("get_col works", {
   expect_identical(get_col("C05"), "05")
   expect_identical(get_col("D11"), "11")
 })
+
+test_that("subset targets works", {
+  dir <- system.file("sample_data", "read_v174", package = "ddpcr")
+  plate <- new_plate(dir)
+  expect_identical(plate %>% subset(targets_ch1 = "t1.fw") %>%
+                     wells_used,
```

Comment:
can you sort the indendation here to be consistent (I think the `wells_used` could go on the previous line)